docs/jailer.md

```diff
@@ -25,6 +26,13 @@ jailer --id <id> \
   the jailer is mostly Firecracker specific.
 - `uid` and `gid` are the uid and gid the jailer switches to as it execs the
   target binary.
+- `cgroup` cgroups can be passed to the jailer to let it set the values
+  when the microVM process is spawned. The `--cgroup` argument must follow this format:
+  `<cgroup_file>=<value>` (e.g cpuset.cpus=0). This argument can be used multiple
+  times to set multiple cgroups. This is useful to avoid providing `sudo` permissions
+  to another process for setting the cgroups before or after the jailer is executed,
+  and for cases when the microVM is configured through a file and starts automatically
```

Comment:
The problem I'm trying to describe is that when a configuration file is used for deploying the microVM it starts automatically as son as the jailer executions ends, so if you want to manually set up a cgroup after jailer execution, it will probably happen with the VM already running. If instead of using a configuration file, the microVM is configured using the Firecracker API, the user will be able to run the jailer (which creates the cgroups hierarchy), create some cgroups manually and after that starts the microVM.

Maybe I can try to rework that part, to try to make it more clear.